tests/Unit/Db/FolderTest.php

```diff
@@ -59,4 +60,28 @@ public function testSerialize()
             ], $folder->jsonSerialize()
         );
     }
+    public function testSetDeletedAt()
+    {
+        $folder = new Folder();
+        $folder->setDeletedAt(15);
+        $this->assertEquals(15, $folder->getDeletedAt());
+    }
+    public function testSetLastModified()
+    {
+        $folder = new Folder();
+        $folder->setLastModified('15');
+        $this->assertEquals('15', $folder->getLastModified());
+    }
+    public function testSetParentId()
+    {
+        $folder = new Folder();
+        $folder->setParentId(15);
+        $this->assertEquals(15, $folder->getParentId());
+    }
```

Comment:
Parent ids (nesting of folders) are currently unused but maybe they'll be supported by the new frontend. (just FYI since I was wondering about it in the DB)

Reply:
You might not see them, but the CLI will already allow nesting